### PR TITLE
updated CorrectedAgeCalaculator to calculate only weeks and days ...

### PR DESCRIPTION
**Pull request by yshauser**
updated CorrectedAgeCalaculator to calculate only weeks and days + some improvements
removed years and months to have only weeks and days in the output
- updated calculation for years to consider premature weeks
- add different output for above 2 years case
- remove the 'overflow' check
- change logic so gestational age above 40 will be consider the same as 40

### CODE REVIEW

File: src/components/CorrectedAgeCalculator.jsx
Comment (relbns) [inline src/components/CorrectedAgeCalculator.jsx]: if you don't need these imports please remove them instead of comment them.

File: src/components/CorrectedAgeCalculator.jsx
Comment (relbns) [inline src/components/CorrectedAgeCalculator.jsx]: same here, please remove code instead of comment it

File: src/components/CorrectedAgeCalculator.jsx
Comment (relbns) [inline src/components/CorrectedAgeCalculator.jsx]: If you want to add relevant correspondence about your changes you can add them like I add these comments inside the pr and not as part of the code.

**Review by yonatankatz — APPROVED** [review]
@yshauser approved from my end, thx!
maybe its worth waiting for Ariel approval as well, as he wrote the code. if you are in a hurry because of the flight - merge this PR, and we will continue this effort if needed.

just remove comments that don't document the current state (as Ariel wrote. comments in the code don't explain the change, but explain the current code. the change should be documented in the PR description / commit message / google doc).
once you merged, we should be able to see the new calculation in "production" (don't worry, its not yet in use).

last thing: it is more common to use a meaningful name for the branch (not just "yh". "yossi" is better, and feature-name is even better, e.g. fix-corrected-age-calculator).
not that important, but a best practice

**Pull request merged**
